.github/actions/wait-for-idle-runner.js

```diff
@@ -0,0 +1,22 @@
+const { Octokit } = require("@octokit/action");
+
+const octokit = new Octokit();
+
+const NBR_ATTEMPTS = 3000;
```

Comment:
What is `NBR`?

Reply:
Number. I changed the const's name again.